firmware/application/apps/soundboard_app.cpp

```diff
@@ -164,7 +166,35 @@ void SoundBoardView::refresh_list() {
 
     // List directories and files, put directories up top
     uint32_t count = 0;
-    for (const auto& entry : std::filesystem::directory_iterator(u"WAV", u"*")) {
+    for (const auto& entry : std::filesystem::directory_iterator(u"/RES/WAV", u"*")) {
```

Comment:
Similar to my suggestion in the SSTV-TX app, I think you could use an outer loop here to go through the two paths, instead of duplicating the code.

Reply:
Moving the duplicated code to a subroutine (and calling it twice - once for each path) would be a good solution too.

Reply:
> Moving the duplicated code to a subroutine (and calling it twice - once for each path) would be a good solution too.

Nice! Thanks! I left you a comment on discord

Reply:
fixed